cartoframes/data/observatory/repository/constants.py

```diff
@@ -0,0 +1,7 @@
+CATEGORY_FILTER = 'category_id'
+COUNTRY_FILTER = 'country_id'
+DATASET_FILTER = 'dataset_id'
+GEOGRAPHY_FILTER = 'geography_id'
```

Comment:
I guess this case is a little bit different since we want to filter by the geography name (i.e.`USA Census Block Group`)